airflow/jobs/scheduler_job.py

```diff
@@ -1344,7 +1344,6 @@ def _run_scheduler_loop(self) -> None:
             conf.getfloat('scheduler', 'clean_tis_without_dagrun_interval', fallback=15.0),
             self._clean_tis_without_dagrun,
         )
-
         for loop_count in itertools.count(start=1):
```

Comment:
```suggestion

        for loop_count in itertools.count(start=1):
```

(Just so we avoid touching this file)